include/sgx/faasm_sgx_native_symbols_wrapper.h

```diff
@@ -5,7 +5,7 @@
 #include <sgx_defs.h>
 
 // Length of used native symbols
-#define FAASM_SGX_NATIVE_SYMBOLS_LEN 25
+#define FAASM_SGX_NATIVE_SYMBOLS_LEN 33
```

Comment:
This is so gross and is bound to be forgotten by the next person who comes to edit this code.

- What actually needs this? Is it something to do with SGX configuration?
- What error comes out if it's wrong? Can we make it more informative?
- When is the error thrown? Can we do it at initialisation time rather than waiting till it's more deeply nested?

Reply:
As discussed offline, I'll do this in a separate PR.